x-pack/test/performance/services/user_actions.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext, FtrService } from '../ftr_provider_context';
+
+interface InputProfile {
+  INPUT_DELAYS: {
+    TYPING: number;
+    MOUSE_CLICK: number;
+  };
+}
+
+export class UserActionsService extends FtrService {
```

Comment:
@spalger can you please let me know if this is how we want to implement this service?

Reply:
I was thinking more along these lines: https://github.com/suchcodemuchwow/kibana/pull/1